### ISSUE TITLE
Canvas video no sound

### ISSUE THREAD

**Issue by candyshien** [newshorts/InlineVideo#5]
How to play video sound on canvas??

**Comment by letitbesd** [comment]
same problem with me. how to play sound and sync with canvas video. thanks.

**Comment by fregante** [comment]
You can use this library, it syncs the audio perfectly: https://github.com/bfred-it/iphone-inline-video

**Comment by happyhj** [comment]
@bfred-it 
But this solution takes traffic twice then just using video tag. right?

**Comment by fregante** [comment]
No, the video is just loaded once by the browser. If you want to make sure, add some cache headers to the server (which is always suggested anyway)
